### PR TITLE
Adding screen shake + Sound Tracks + Sound Effects

### PR DESCRIPTION
**Pull request by ultimateprogramer**
Hi @britzl . I am a recent Defold Enthusiast and have been doing some private game development. I was fascinated by GB Rausers but wanted to contribute functionality to it over time. In this specific pull request I place in:

* Screen shake: When the player is hit by a bullet or when a player shoots.
* Dynamically fitting GUI and other screens when the window is resized.

I hope to work to add some exciting game features over time.

### CODE REVIEW

**Comment by britzl** [comment]
I haven't opened this project in ages and when I did I immediately noticed some things I would have done differently or that needed fixing:

* The custom render script is no longer needed
* The special font material is no longer needed
* The camera handling could be simplified

I decided to make theses changes and merge them to the master branch:

https://github.com/britzl/gbrausers/pull/2

**Comment by britzl** [comment]
The camera shake was a nice touch, but I found it a bit annoying with the shake when you fired bullets as well. It became a bit too much.

But if you'd like to keep it in your version then please by all means make a fork on GitHub and apply the changes there instead!

**Comment by ultimateprogramer** [comment]
@britzl yeah I had realized there were some new conventions of doing things and that also `ludobits` was updated also. Thanks for this, I think I will first of all merge any new changes you have made to update them and also remove the camera shake when firing bullets and only retain those that happen when the player is hit. Then re-align everything again for a proper merge with your new request to remove the annoying bit :smile: 

My intention is to eventually develop this game to demonstrate how a proper production game has, including maybe even have some sound assets, etc. To serve as an inspiration for both myself and other devs as to how proper game setups can be done. But I guess we can take that a step at a time :smile:

**Comment by ultimateprogramer** [comment]
@britzl I have merged changes from upstream and done away with the shoot screen shake. Only retained the when hit screenshake

File: camera/camera.script [hunk 1]
```diff
@@ -0,0 +1,34 @@
+go.property("aspect_ratio", 1.0)
```
Comment (britzl) [inline camera/camera.script]: This file is not used. It should be removed.

File: camera/camera.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
+components {
```
Comment (britzl) [inline camera/camera.go]: This file is not used. It should be removed.

**Review by britzl — CHANGES_REQUESTED** [review]

File: camera/camera.go (hunk 2 above)
Comment (ultimateprogramer) [reply camera/camera.go]: Done. Camera object removed

**Comment by ultimateprogramer** [comment]
Camera object removed

**Comment by ultimateprogramer** [comment]
@britzl I have added sounds to the game over the weekend in 2 areas:

* Sound tracks for startup menu and game play scene.
* Sound effect for shooting for the player object.

**Review by britzl — CHANGES_REQUESTED** [review]
@ultimateprogramer I'm happy to link to your fork with the improvements you've made but I would prefer to keep the master repo (ie mine) unmodified except for the changes to the README.md and the change to use the default render script and camera component.

The camera shake effect, sounds etc should be in your improved version/fork

**Comment by ultimateprogramer** [comment]
@britzl totally OK with me. Should we therefore dismiss this PR? :)

**Comment by britzl** [comment]
Yes, that might be best. And submit a new one with an updated readme that links to your fork with your improvements.

**Pull request closed without merging**